pylint/checkers/imports.py

```diff
@@ -444,29 +442,11 @@ def __init__(self, linter: Optional["PyLinter"] = None) -> None:
         self._site_packages = self._compute_site_packages()
 
     @staticmethod
-    def _compute_site_packages():
-        def _normalized_path(path):
+    def _compute_site_packages() -> Set[str]:
+        def _normalized_path(path: str) -> str:
             return os.path.normcase(os.path.abspath(path))
 
-        paths = set()
-        real_prefix = getattr(sys, "real_prefix", None)
-        for prefix in filter(None, (real_prefix, sys.prefix)):
-            path = sysconfig.get_python_lib(prefix=prefix)
-            path = _normalized_path(path)
-            paths.add(path)
-
-        # Handle Debian's derivatives /usr/local.
-        if os.path.isfile("/etc/debian_version"):
-            for prefix in filter(None, (real_prefix, sys.prefix)):
-                libpython = os.path.join(
-                    prefix,
-                    "local",
-                    "lib",
-                    "python" + sysconfig.get_python_version(),
-                    "dist-packages",
-                )
-                paths.add(libpython)
-        return paths
+        return {_normalized_path(path) for path in astroid.modutils.STD_LIB_DIRS}
```

Comment:
```suggestion
        return {_normalized_path(path) for path in astroid.modutils.EXT_LIB_DIRS}
```
If these should be the paths to the `site-packages` dirs, it should be `EXT_LIB_DIRS` instead.
See https://github.com/PyCQA/astroid/pull/1322

Reply:
Thanks, bad mistake. That was the PR I was looking at yesterday when I started down this path, and then I mistook the recent astroid changes for being relevant. 🤕